app/assets/javascripts/components/campaign/campaign.jsx

```diff
@@ -29,24 +26,18 @@ export const Campaign = createReactClass({
     if (this.props.campaign.loading) {
       return <div />;
     }
-    if (window.location.href.match(/overview/)) {
-      return <CampaignOverviewHandler {...this.props} />;
-    }
+
     return (
-      <div>
-        <CampaignNavbar
-          campaign={this.props.campaign}
-        />
+      <div className="container">
+        <CampaignOverviewHandler {...this.props} />
```

Comment:
In terms of naming conventions, the 'overview handler' should be the component that specifically handles the `overview` tab, as opposed to shared component that handles the stats for all the tabs.